### ISSUE TITLE
Splitters broken again

### ISSUE THREAD

**Issue by Georgehe4** [dart-lang/dart-pad#594]
Embed splitters do not work - bug with Chrome version 44.0.
<img width="716" alt="screen shot 2015-07-27 at 1 43 26 pm" src="https://cloud.githubusercontent.com/assets/3150228/8906320/8bb519e8-3465-11e5-8d2b-e9594359d0fe.png">

**Comment by Georgehe4** [comment]
<img width="1407" alt="screen shot 2015-07-27 at 1 37 04 pm" src="https://cloud.githubusercontent.com/assets/3150228/8906256/f8eaec78-3464-11e5-8bd8-03651a9c2b1a.png">
<img width="1410" alt="screen shot 2015-07-27 at 1 37 11 pm" src="https://cloud.githubusercontent.com/assets/3150228/8906255/f8e85ecc-3464-11e5-9e1c-e40757a1574a.png">

**Comment by Georgehe4** [comment]
<img width="1411" alt="screen shot 2015-07-27 at 3 37 09 pm" src="https://cloud.githubusercontent.com/assets/3150228/8908711/775c80a2-3475-11e5-9345-46d9e48ddd16.png">
<img width="594" alt="screen shot 2015-07-27 at 3 37 20 pm" src="https://cloud.githubusercontent.com/assets/3150228/8908712/775dd47a-3475-11e5-8101-c3af6ec71e3b.png">

**Comment by Georgehe4** [comment]
<img width="1415" alt="screen shot 2015-07-27 at 7 09 46 pm" src="https://cloud.githubusercontent.com/assets/3150228/8913861/8f1fde32-3493-11e5-81c0-ce6739231134.png">
Found the issue. Default values are all set to "auto" instead of pixel values (or a null value), causing the calculation for minimum width to fail since we were parsing a value expected to be an integer, not a word. I've updated accordingly and will send in a PR.
